browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1718,6 +1728,90 @@ void RewardsDOMHandler::GetBalanceReport(const base::ListValue* args) {
                      year));
 }
 
+void RewardsDOMHandler::OnGetMonthlyReport(
+    const uint32_t month,
+    const uint32_t year,
+    const brave_rewards::MonthlyReport& report) {
+  if (!web_ui()->CanCallJavascript()) {
+    return;
+  }
+
+  base::Value data(base::Value::Type::DICTIONARY);
+  data.SetIntKey("month", month);
+  data.SetIntKey("year", year);
```

Comment:
Recommend we do const strings for new keys

Reply:
please log an issue and we will do it in one go in this file